openmp/runtime/src/z_Linux_util.cpp

```diff
@@ -66,6 +66,12 @@
 #include <sys/types.h>
 #include <sys/sysctl.h>
 #elif KMP_OS_SOLARIS
+#if defined(__LP64__)
+#define _STRUCTURED_PROC 1
+#include <sys/procfs.h>
```

Comment:
(GH is driving me nuts: lost part of my comments for the second time ;-()

Why do things this way?  The documented way to access `procfs` is to just include `<procfs.h>` (which deals with `_STRUCTURED_PROC` internally).

Reply:
I did not know procfs.h did all, handy.